docs/modules/ROOT/pages/documentation/payara-micro/appendices/cmd-line-opts.adoc

```diff
@@ -55,6 +55,8 @@ service is active.
 |`--domainconfig <file-path>`
 |Overrides the server configuration with an alternative _domain.xml_ file.
 |If not set, the domain.xml contained in the Payara Micro JAR is used.
```

Comment:
```suggestion
If not set, the domain.xml contained in the Payara Micro JAR is used.
```
Remove `|` so that both lines end up in the same table cell